project_task_material_stock/models/account_analytic_line.py

```diff
@@ -1,23 +1,20 @@
 # Copyright 2019 Valentin Vinagre <valentin.vinagre@qubiq.es>
+# Copyright 2020 jeo Software Jorge Obiols <jorge.obiols@gmail.com>
```

Comment:
I'm not sure it's well received to insert copyrights claims when you just reformat a file like that...